src/ImageGallery.jsx

```diff
@@ -77,7 +78,8 @@ export default class ImageGallery extends React.Component {
     renderPlayPauseButton: PropTypes.func,
     renderFullscreenButton: PropTypes.func,
     renderItem: PropTypes.func,
-    stopPropagation: PropTypes.bool
+    stopPropagation: PropTypes.bool,
+    className: PropTypes.string
```

Comment:
Lets call this something else, maybe `additionalClass` ?

see https://github.com/airbnb/javascript/blob/master/react/README.md#naming